src/data/docs/tour/README.md

```diff
@@ -284,7 +284,7 @@ square x = x * x
 
 use test
 
-test> square.tests.ex1 = check (square 4 == 16)
```

Comment:
`check (square 4 == 16)` ought to work, so let's get to the bottom of that before updating the docs.

Reply:
The ucm UI that opens show in the package base then test that the operator `===` and not `==` exist...

Reply:
![image](https://user-images.githubusercontent.com/6291619/125923099-9064264c-c50b-4ab5-8363-9b921f359605.png)